runtime/oti/vmconstantpool.xml

```diff
@@ -358,6 +358,10 @@
 	<!-- Security manager check -->
 	<staticfieldref class="java/lang/System" name="security" signature="Ljava/lang/SecurityManager;"/>
 
+	<!-- Access to Direct Byte Buffer native memory allocation data -->
+	<staticfieldref class="java/nio/Bits" name="count" signature="Ljava/util/concurrent/atomic/AtomicLong;"/>
+	<staticfieldref class="java/nio/Bits" name="reservedMemory" signature="Ljava/util/concurrent/atomic/AtomicLong;"/>
+
```

Comment:
You'll also need an entry for the `value` field in AtomicLong.

Reply:
Done. It was faster to make a new PR.

New PR: https://github.com/eclipse/openj9/pull/1508